workspaces/spectacle/src/graphql/schema.ts

```diff
@@ -106,6 +106,24 @@ type HttpBody {
   isRemoved: Boolean
 }
 
+"""
+HttpQuery - query parameters associated with a request
+"""
+type HttpQuery {
+  id: ID!
+
+  # The query parameter key
+  key: String!
+
+  # Reference to the shape for this query parameter
+  rootShapeId: String!
+
+  # Is the query removed
+  isRemoved: Boolean!
+
+  changes(sinceBatchCommitId: String): ChangesResult!
```

Comment:
I _think_ this is necessary for the HTTP query since we also have `key`, but I'm not 100% sure here - we'll be able to find out as we add in more